dist/docker/Readme.md

```diff
@@ -1,68 +1,71 @@
-# Docker Container Name
+# qBittorrent-nox Docker Image
 
-This Dockerfile allows you to build a qBittorrent-nox container
+This Dockerfile helps you to build a docker image containing qBittorrent-nox
```

Comment:
`Help` is better suited to something that is not directly involved in the action, but Dockerfile is involved, so I would prefer to leave `allow`.